Assets/MixedRealityToolkit.SDK/Experimental/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -686,10 +689,10 @@ private void HandleManipulationStarted()
             if (rigidBody != null)
             {
                 wasKinematic = rigidBody.isKinematic;
-                rigidBody.isKinematic = true;
+                rigidBody.isKinematic = false;
```

Comment:
Do we want to make this new behavior optional? Or is the presence of the rigidbody now the "opt-in" for this feature?

Reply:
That works for me!